app/components/subnav_component/nav_item_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<li class="app-subnav__list-item<%= current_page?(path, check_parameters: true) ? " app-subnav__list-item--selected" : "" %>">
+<li class="app-subnav__list-item<%= current_page?(path) ? " app-subnav__list-item--selected" : "" %>">
```

Comment:
This might be easier to read as a method in the `NavItemComponent`.

```def
def subnav_list_item_classes
  class_names(
    "app-subnav__list-item"
    "app-subnav__list-item--selected" => current_page?(path)
  )
end
```

```
tag.li(class: subnav_list_item_classes) do
  ...
end
```